testdata/p4_16_errors_outputs/issue3623-2.p4-stderr

```diff
@@ -0,0 +1,6 @@
+issue3623-2.p4(8): [--Werror=type-error] error: 'v' with type 'e' cannot be compared to '4w1' with type 'bit<4>'
```

Comment:
This error message are a little vague. Any way to make it more clear that this concerns a (ser)enum?

Reply:
Removing 'toString' will show better error messages. Just pushed this change.